### DISCUSSION TITLE
Wireless File Manager has been updated!

### DISCUSSION THREAD

**Discussion by visnkmr** [visnkmr/visnkmr#1] [Announcements]
Hey,
We have pushed an update to Amazon Appstore today. It may take 24 hours or more for Appstore to complete their review process; We will update here once its live. 

If you have any features, UI changes you would like to see in the app please do tell me about it.

**Comment by jaxposse** [comment]
During transfer my file was compressed and needs a password to unzip which I do not have. How do I avoid  file compression or is there a standard password for unzipping compressed files?

**Reply by visnkmr** [reply]
Hey,
wireless file manager does not compress or archive the file during transmission.

**Comment by jaxposse** [comment]
Thanks for the response. I can now look elsewhere for the solution. Great
app by the way.
